src/components/pages/home/swag/swag.jsx

```diff
@@ -19,66 +24,122 @@ import tShirt from './images/t-shirt.jpg';
 const title = 'Swag list';
 const swagList = [tShirt, mug, sticker, bag, box, hoodie, tShirt2, sticker2];
 
-const Swag = () => (
-  <section className="safe-paddings relative py-26 md:py-20 sm:py-16 xs:py-12" id="swag">
-    <div className="container relative z-10">
-      <SectionHeading className="text-center">{title}</SectionHeading>
-      <p className="mx-auto mt-10 max-w-[716px] text-center text-lg md:mt-8 md:text-[18px] sm:mt-6 sm:text-base">
-        Join a squad, Contribute code and win Swag. Simple.
-        <br />
-        Swag list will be{' '}
-        <span className="relative before:absolute before:-left-1.5 before:-z-10 before:h-[30px] before:w-[318px] before:bg-primary-1">
-          updated soon
-        </span>
-      </p>
+const Swag = () => {
+  const [popup, setPopup] = useState(false);
+  const [content, setContent] = useState('');
+
+  const handleClose = () => {
+    setPopup(false);
+  };
+
+  return (
+    <section
+      className="safe-paddings relative select-none py-26 md:py-20 sm:py-16 xs:py-12"
+      id="swag"
+    >
+      <div className="container relative z-10">
+        <SectionHeading className="text-center">{title}</SectionHeading>
+        <p className="mx-auto mt-10 max-w-[716px] text-center text-lg md:mt-8 md:text-[18px] sm:mt-6 sm:text-base">
+          Join a squad, Contribute code and win Swag. Simple.
+          <br />
+          Swag list will be{' '}
+          <span className="relative before:absolute before:-left-1.5 before:-z-10 before:h-[30px] before:w-[318px] before:bg-primary-1">
+            updated soon
+          </span>
+        </p>
+
+        <ul className="grid-gap relative mt-20 grid grid-cols-4 md:mt-16 sm:mt-10 sm:grid-cols-2">
+          {swagList.map((item, index) => (
```

Comment:
Instead of going over the swagList array and using index to fetch items, you might want to add the image in the desc array file itself and just iterate over that array.

In that way you won't have to import image in different file and also you want have to rely on index position to get title and description.